generators/client/templates/angular/src/test/javascript/jest.conf.js.ejs

```diff
@@ -25,3 +25,12 @@ module.exports = {
     rootDir: '../../../',
     testURL: 'http://localhost/'
 };
+
+function addAliasesFromTypescriptConfig(initialAliases) {
```

Comment:
can we write it like other comment? Also, can we simplify the usage of regex?

Reply:
Function rewritten as in Webpack part and regex usage simplified.